python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -493,7 +495,9 @@ def set_task(self, task):
         return server, tracker
 
 
-def _build_func_common(measure_input, runtime=None, check_gpu=None, build_option=None):
+def _build_func_common(
+    measure_input, runtime=None, check_gpu=None, build_option=None, check_hexagon=None
```

Comment:
Probably we could pass all check arguments (such as `check_gpu` and `check_hexagon`) in a dictionary or list? In this case it won't be necessary to change function signature every time when we want to add new check

Reply:
done.